jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -162,6 +162,10 @@ function configure_EJB_Timer_datasource {
 function set_url {
     local prefixedUrl=$(find_env "${1}_URL")
     url=$(find_env "${1}_XA_CONNECTION_PROPERTY_URL" "${prefixedUrl}")
+     # RHPAM-4710 EJB_TIMER Ds Auto creation does not recognizes Url XA property
+    if [ "${url}x" = "x" ]; then
```

Comment:
is this needed?
the addition on line 200 should be enough to handle it.
if you do this there will *_URL and *_Url set together.

Reply:
If we don't want that RHPAM_URL is equal to RHPAM__XA_CONNECTION_PROPERTY_Url we can remove this

Reply:
It is not needed, RHPAM__XA_CONNECTION_PROPERTY_Url  should not be set if RHPAM_URL is there.